site2/website-next/docs/reference-cli-broker-tool.md

```diff
@@ -1,36 +0,0 @@
----
-id: reference-cli-broker-tool
-title: Pulsar CLI tools - broker-tool
-sidebar_label: "cli-broker-tool"
----
-
-## `broker-tool`
-
-The `broker- tool` is used for operations on a specific broker.
-
-Usage
-
-```bash
-broker-tool command
```

Comment:
It seems that we remove this utility for a long time.

cc @merlimat @codelipenghui @nicoloboschi do you have ideas of the specific commit we remove this tool? I'm thinking of which versions of docs we should do the corresponding changes.

Reply:
OK I found https://github.com/apache/pulsar/pull/18051#issuecomment-1280297019.